pkg/server/errors/errors.go

```diff
@@ -121,10 +119,6 @@ func InvalidAuthorizationModelInput(err error) error {
 // Use `public` if you want to return a useful error message to the user.
 func HandleError(public string, err error) error {
 	switch {
-	case errors.Is(err, storage.ErrTransactionalWriteFailed):
-		return status.Error(codes.Aborted, err.Error())
-	case errors.Is(err, storage.ErrInvalidWriteInput):
-		return WriteFailedDueToInvalidInput(err)
```

Comment:
no longer relevant?

Reply:
No, since the only API that can return these errors is Write API, and I moved this code to the Write command.